build/local_builder.go

```diff
@@ -256,7 +257,7 @@ func (l *LocalBuilder) execLayer(layer *v1.Layer, tempTarget string) error {
 			dist = filepath.Join(tempTarget, strings.Fields(layer.Value)[1])
 		} else {
 			// src is file
-			dist = filepath.Join(tempTarget, strings.Fields(layer.Value)[1], strings.Fields(layer.Value)[0])
+			dist = filepath.Join(tempTarget, strings.Fields(layer.Value)[1], path.Base(strings.Fields(layer.Value)[0]))
```

Comment:
i think we should keep the same principle with docker COPY. that is mean:  the <src> path must be inside the context of the build. and the <dest> path must be an absolute path inside the container.

Reply:
as you say,
```
Copy sth /etc/sth
```
in sealer context is  absolute path /etc/sth

Reply:
absolute path inside the container is not related  to Copy source path, is related to Copy destination path